content/sensu-go/6.5/operations/deploy-sensu/install-sensu.md

```diff
@@ -220,6 +231,13 @@ sensu-backend init
 
 {{< /language-toggle >}}
 
+To initialize with your admin API key, replace `<your_api_key>` with the API key you want to use:
+
+{{< code shell >}}
+export SENSU_BACKEND_CLUSTER_ADMIN_API_KEY=<your_api_key>
+sensu-backend init
+{{< /code >}}
+
```

Comment:
@echlebek what does this mean for step 4. Open the web UI? If you used the API key init option instead of username and password with Ubuntu/Debian or RHEL/CentOS, what do you use to log in to the web UI?

Reply:
API Keys are keys for _application programming interfaces_, not interactive applications like the Web UI. There is no way to use them with the Web UI.

Reply:
well, but what I'm asking is how they get a username and password to log into the web UI if they've used the API key to initialize. The way we present it in the docs right now, that initialize step is where you specify the username and password. If you don't do that because you're using API key to initialize, does that mean the admin user automatically gets the default username and password?

Reply:
Oh, no. The username and password are still required to create the user.

Reply:
I'm starting to think I misunderstand this feature :D

Reply:
API keys are simply a secondary access method for users that already have a username and password.